db/migrate/20210510161040_remove_export_content_views_perm.rb

```diff
@@ -0,0 +1,20 @@
+class RemoveExportContentViewsPerm < ActiveRecord::Migration[6.0]
+  def permission_map
+    {
+      Permission.find_by(name: :export_content_views) => Permission.find_by(name: :export_content)
```

Comment:
Thought we were moving this to an upgrade script?

Reply:
Working on the script.